docs/src/openapi-v1.yaml

```
@@ -564,6 +564,59 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/RuntimeSpec'
+  /pallets/{palletId}/storage:
+    get:
+      tags:
+      - pallets
+      summary: Get a list of storage items for a pallet.
+      description: Returns a list of storage item metadata for storage items of the
+        specified palletId.
+      parameters:
+      - name: palletId
+        in: path
+        description: 'Name or index of the pallet to query the storage of. Note: pallet name aligns with
+          pallet name as specified in runtime metadata.'
```

"aligns with", does that mean it is the same as the name specified in the runtime metadata? 

```suggestion
        description: 'Name or index of the pallet to query the storage of. Note: the pallet name must match what is
          specified in the runtime metadata.'
```

---

Sometimes the description is surrounded by `'` and sometimes not. Why is that?

---

I use the `'` when there is a character within the string that needs to be escaped. In this case the `:` after `Note:` was giving me trouble since it thought `Note:` was a yaml key.